pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -44,3 +103,24 @@ var (
 		},
 	}
 )
+
+func RewriteNodeIPs(nodeList *corev1.NodeList) *metadataOptions {
+	nodes := map[string]int{}
+	for i, node := range nodeList.Items {
+		nodes[node.Name] = i
+	}
+	return &metadataOptions{
+		rewriteSecret: func(secret *corev1.Secret) {
+			for nodeName, masterID := range nodes {
+				name := strings.ReplaceAll(secret.Name, nodeName, fmt.Sprintf("master-%d", masterID))
```

Comment:
```suggestion
				name := strings.ReplaceAll(secret.Name, nodeName, fmt.Sprintf("<master-%d>", masterID))
```

This makes it clear that we have substitution happening when people do things like read generated docs